public/js/main.js

```diff
@@ -101,3 +85,42 @@ function renderToolbox() {
     document.querySelector("#mount")
   );
 }
+
+if(process.env.NODE_ENV !== "DEVTOOLS_PANEL") {
```

Comment:
My gut feeling is a little different. This is a build flag, and config files (to me) are runtime flags. The risk with moving this to a runtime flag is that somehow the wrong config is used to generate the bundle and when we automatically publish this into Firefox it doesn't work. `webpack.config.devtools.js` wants to unconditionally build a bundle for Firefox.

The other advantage is that it removes a step from the whole process of generating the Firefox bundle, so other people that do it just run `webpack --config webpack.config.devtools.js` and they get it. They don't need to muddle with config.

Another example: locally I'm developing something, then I want to test it in the Firefox devtools. I get the feature working and all I have to do is run that webpack command. I don't need to go into my config to switch the flag, generate it, and then switch it back. If I forget to switch it back I can't just run it locally.

Perhaps I like build-time flags more than config flags, because it seems simpler to coordinate local development.

Maybe there's a way to still specify this flag at build-time from webpack, but actually change the config json value? We could do this with things like `nconf` but I'm not sure the best way to set it here.